Dockerfile

```diff
@@ -14,12 +14,13 @@ ENTRYPOINT [ "/sbin/tini", "--" ]
 FROM base AS build-base
 WORKDIR /build
 COPY package.json package-lock.json prisma ./
-RUN npm ci --omit=dev --frozen-lockfile \
- && npx prisma generate
+RUN npm ci --include=dev --frozen-lockfile 
+RUN npx prisma generate
```

Comment:
It's fine to drop `--omit` but why split it into two layers (two `RUN` directives)? AFAIK you have to run generate every time you run `ci`. Or am I missing something?